Directory.Packages.props

```diff
@@ -150,4 +150,4 @@
     <PackageVersion Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="$(MicrosoftExtensionsPackagesVersion)" />
     <!-- Microsoft.Extensions.Http.Resilience is not here because it diverged from the common Microsoft.Extensions versioning. -->
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
```suggestion
</Project>

```

Reply:
I previously re-added the line break, but it's removed by VS when doing a NuGet update, so I wouldn't bother always keeping it there manually just for the sake of it.